jmeter/DataFixtures/LoadNodeData.php

```diff
@@ -81,6 +105,11 @@ public function generateSimpleNode($name, $language, ObjectManager $manager, $ht
         $node->addBlock($contentBlock);
         $node->setInMenu(true);
 
+        $routes = $this->updateRoute->createForNode($node);
```

Comment:
I would rather persist the node before running the updateRoute manager (because of the id).

Or does it work perfectly like that ?